packages/runtime/src/lib/itxn.ts

```diff
@@ -72,6 +72,10 @@ export function setInnerTxField (
     const accountState = interpreter.getAccount(assertedVal, line);
     txValue = Buffer.from(decodeAddress(accountState.address).publicKey);
   }
+  // begin supported RekeyTo form teal version 6
+  if (field === 'RekeyTo' && interpreter.tealVersion >= 6) {
+    txValue = op.assertAlgorandAddress(val, line);
+  }
```

Comment:
Can we put the fields into an array/set or something? What if we need to add more fields for more versions?